src/audioPlayer.js

```diff
@@ -0,0 +1,96 @@
+// Instantiate the PLAYER variable in the Webflow page's head since it is needed across multiple scripts.
+PLAYER = document.querySelector('[data-element=audio-player]')
+PLAYER.querySelector('source').src = '0'
+
+const AUDIO_TOGGLES = document.querySelectorAll('[data-element=player-toggle]')
+
+AUDIO_TOGGLES.forEach((podcastTrigger) => {
```

Comment:
Naming nit: I am not sure what `podcast` means in this context. Is it the play button?

Reply:
Yeah this was some legacy naming, I'll do a pass to clean this up!